### ISSUE TITLE
mac 上使用  backend = transformers  报错

### ISSUE THREAD

**Issue by kenneth-bro** [opendatalab/MinerU#2977]
### 🔎 Search before asking | 提交之前请先搜索

- [x] I have searched the MinerU [Readme](https://github.com/opendatalab/MinerU) and found no similar bug report.
- [x] I have searched the MinerU [Issues](https://github.com/opendatalab/MinerU/issues) and found no similar bug report.
- [x] I have searched the MinerU [Discussions](https://github.com/opendatalab/MinerU/discussions) and found no similar bug report.

### 🤖 Consult the online AI assistant for assistance | 在线 AI 助手咨询

- [x] I have consulted the [online AI assistant](https://deepwiki.com/opendatalab/MinerU) but was unable to obtain a solution to the issue.

### Description of the bug | 错误描述

<img width="1096" height="154" alt="Image" src="https://github.com/user-attachments/assets/547d625a-c242-45b1-8659-71604388f596" />

2025-07-11 09:52:44.152 | INFO     | mineru.backend.vlm.predictor:get_predictor:110 - get_predictor cost: 1.64s
Predict:   0%|                                                                 | 0/13 [00:00<?, ?it/s]The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

就这样不动了

### How to reproduce the bug | 如何复现

使用fastapi的方式启动服务并请求。
curl -X 'POST' \
  'http://192.168.48.10:8999/file_parse' \
  -H 'accept: application/json' \
  -H 'Content-Type: multipart/form-data' \
  -F 'return_middle_json=false' \
  -F 'return_model_output=false' \
  -F 'return_md=true' \
  -F 'return_images=false' \
  -F 'end_page_id=99999' \
  -F 'parse_method=auto' \
  -F 'start_page_id=0' \
  -F 'lang_list=en' \
  -F 'output_dir=./output' \
  -F 'server_url=string' \
  -F 'return_content_list=false' \
  -F 'backend=transformers' \
  -F 'table_enable=true' \
  -F 'files=@prepped.pdf;type=application/pdf' \
  -F 'formula_enable=true'

### Operating System Mode | 操作系统类型

MacOS

### Operating System Version| 操作系统版本

15.1

### Python version | Python 版本

3.12

### Software version | 软件版本 (mineru --version)

2.0.x

### Device mode | 设备模式

mps
